addons/purchase/wizard/purchase_batch_bills.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
```

Comment:
@yhu-odoo Update: You can replace the wizard by a single action that create the account moves and open them.

Reply:
@amoyaux  I used a server action to implement that. I don't if that is the best way, you can take a look.

Reply:
@amoyaux I found when use a server action, if we violate some constrains, eg. using a purchase journal without any account. We won't get a regular user error, instead, we will have a traceback. Is there any other way to implement it? Thanks.

Reply:
@amoyaux it seems if we violate a sql constraint when executing codes in a  server action, we can only get a traceback instead of a validation error. Should we go back to the previous wizard solution?